chemistry-lab-compounds.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: default
+title: Chemistry Lab Compounds
+---
+
+# TARDIS Chemistry Lab Compounds
+
+The [Chemistry Lab](chemistry-lab) can produce compounds, typically used in crafting Products!
+
+_This page only shows compounds made in the chemistry lab, to view products, go to [chemistry lab compounds](chemistry-lab-products), to view how to use the chemistry lab, go to the [chemistry lab page](chemistry-lab). To view a list elements, go to the [elements page](elements)_
+
+All compounds are inherently useless on their own (with the exception of the vanilla items and their vanilla uses). They must be combined with other compounds, [elements](elements), or items to make [products]((chemistry-lab-products))!
+
+TIP: `/tchem` and `/tchemistry` are valid aliases of `/tardischemistry`
+
+## Resource Pack
+
+**_The compounds are best viewed with the [resource pack](https://github.com/eccentricdevotion/TARDIS-Resource-Pack)._**
+
+All images use the TARDIS-Resource-Pack, and your experience will be much better with it!
+
+## Table of Contents
```

Comment:
Can we use a table here and make the content two columns (like site map but without the ABCs)?

Reply:
Likewise for other TOCs

Reply:
oop, I sorted by letter anyway